recipes-graphics/waylandpp/waylandpp_0.2.8.bb

```diff
@@ -35,19 +35,19 @@ EXTRA_OECMAKE_append_class-target = " \
     -DCMAKE_EXE_LINKER_FLAGS="-Wl,--enable-new-dtags" \
     "
 
-FILES_${PN} = " \
+FILES:${PN} = " \
     ${libdir}/libwayland*.so* \
     "
 
-FILES_${PN}-dev  = " \
+FILES:${PN}-dev  = " \
     ${includedir}/* \
     ${libdir}/pkgconfig/* \
     ${libdir}/cmake/waylandpp/* \
     ${datadir}/waylandpp/protocols/* \
     "
 
-FILES_SOLIBSDEV = ""
-INSANE_SKIP_${PN} += "dev-so"
+FILES:SOLIBSDEV = ""
```

Comment:
Not sure there, I think `FILES:SOLIBSDEV` is just a variable.

Reply:
Also thought about that. To me SOLIBSDEV looks like a package-name, but i'm unsure too. I am even unsure if that line is needed at all